RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -98,6 +98,7 @@ public struct CustomerCenterView: View {
 #endif
                 self.trackImpression()
             }
+            .modifier(CustomerCenterActionPreferenceConnector(viewModel: viewModel))
```

Comment:
I wonder if it's more precise to apply this just to `content` rather than at the body top level